validation/utils.py

```diff
@@ -0,0 +1,7 @@
+import re
+from typing import Tuple
+
+
+def get_first_found_or_none(regex: str, text: str) -> Tuple[bool, str]:
+    matched = re.compile(regex).search(text)
+    return (True, matched.groups()[0],) if matched else (False, '',)
```

Comment:
Do you really need to return a boolean? `''` resolves to `False` anyway...

Also, a more accurate name would be `get_first_match_found_or_empty`